<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  #1F1F1F;
padding_bottom: 10px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>114</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #483D8B;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: white;
font-size:26px;
</string>
       </property>
       <property name="text">
        <string>Номер страницы</string>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLineEdit" name="numberLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;
background-color:white;
font-size: 15px;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>114</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #483D8B;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: white;
font-size:26px;
</string>
       </property>
       <property name="text">
        <string>Тип элементов</string>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
font-size: 15px;
</string>
       </property>
       <item>
        <property name="text">
         <string>диоды</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>транзисторы</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>конденсаторы</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>резистры</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>600</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #483D8B;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="autoLaunchBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #1f1f1f;
color: white;
font-size: 15px;
padding: 20px;
border-radius: 10px;
</string>
       </property>
       <property name="text">
        <string>Запустить автоматически</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>10</y>
      <width>334</width>
      <height>234</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #483D8B;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;
font-size:26px;
</string>
          </property>
          <property name="text">
           <string>Chrome driver</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QPushButton" name="driverBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #1f1f1f;
color: white;
font-size: 15px;</string>
          </property>
          <property name="text">
           <string>Запустить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_6" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;
font-size:26px;
</string>
          </property>
          <property name="text">
           <string>Parser</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QPushButton" name="parserBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #1f1f1f;
color: white;
font-size: 15px;</string>
          </property>
          <property name="text">
           <string>Запустить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
